roycemoses/cs179n-project
# Aavit004/picking up potions

To test OpeningCutScene:
- Start with MainMenuBG and press the new game Button. That will show you opening cutscenes and you end up in Homebase. 

To test the Neighborhood and Home go to the scenes and press play. Then you can see the walls and stuff work.
